### PR TITLE
[frontend] add fred case study

### PR DESCRIPTION
**Pull request by Fbasham**
### Description

Fred gets a case study. 

List of proposed changes:
- modify fred page
- add translations and images for mobile/desktop
- modify some existing styles on accessibility containers

### CODE REVIEW

File: frontend/src/pages/learn/case-studies/fred.tsx [hunk 1]
```diff
@@ -1,23 +1,181 @@
...
 import { LearnPageLayout } from '../../../components/LearnPageLayout'
+import theme from '../../../theme'
 import { getDCTermsTitle } from '../../../utils/seo-utils'
 
 const Fred: FC = () => {
   const { t, i18n } = useTranslation('learn/case-studies/fred')
   const en = i18n.getFixedT('en', 'learn/case-studies/fred')
   const fr = i18n.getFixedT('fr', 'learn/case-studies/fred')
 
+  let mobile = useMediaQuery(theme.breakpoints.down('sm'))
```
Comment (sebastien-comeau) [inline frontend/src/pages/learn/case-studies/fred.tsx:23]: Should be `const` and not `let`.

File: frontend/src/pages/learn/case-studies/fred.tsx [hunk 2]
```diff
@@ -1,23 +1,181 @@
...
+            text: t('breadcrumbs.learn.text'),
+          },
+        ]}
+      >
+        <NextSeo title={t('header')} />
+        
+        <h2 id="key-takeaways" className="h2 !mt-0">
+          {t('key-takeaways.heading')}
+        </h2>
+        <ul className='space-y-2'>
```
Comment (sebastien-comeau) [inline frontend/src/pages/learn/case-studies/fred.tsx:82]: Should use MUI List component as mentioned in PR #182

File: frontend/src/pages/learn/case-studies/fred.tsx (hunk 1 above)
Comment (Fbasham) [reply frontend/src/pages/learn/case-studies/fred.tsx]: fixed

File: frontend/src/pages/learn/case-studies/fred.tsx (hunk 2 above)
Comment (Fbasham) [reply frontend/src/pages/learn/case-studies/fred.tsx]: fixed

**Review by sebastien-comeau — APPROVED** [review]

**Pull request merged**